src/views/Transitioner.js

```diff
@@ -155,7 +155,11 @@ class Transitioner extends React.Component<*, Props, State> {
     const { timing } = transitionSpec;
     delete transitionSpec.timing;
 
-    const animations = indexHasChanged
+    const toValue = nextProps.navigation.state.index;
+    const positionHasChanged = position.__getValue() !== toValue;
```

Comment:
If you still think it's needed, I'll make the change, I just don't want to make it complicated